apps/finance/app/src/lib/token-utils.js

```diff
@@ -38,3 +44,30 @@ export const tokenDataFallback = (tokenAddress, fieldName, networkType) => {
   }
   return fallbacksForNetwork.get(addressWithoutChecksum)[fieldName] || null
 }
+
+export async function getTokenSymbol (app, address){
+  const tokenAbi = [].concat(tokenBalanceOfAbi,tokenDecimalsAbi,tokenSymbolAbi)
+  const token = app.external(address,tokenAbi)
+  const tokenSymbol = await token
+    .symbol()
+    .first()
+    .toPromise()
+  if (tokenSymbol){
+    return tokenSymbol
+
+  }else{
```

Comment:
Oh yes sorry I should have been more explicit, yes I am only talking about removing the `else` and not the block itself, because the `if` block returns immediately.

Basically converting this kind of structure:

```js
if (a) {
  return 'a'
} else {
  /* do stuff */
  if (b) {
    return 'b'
  } else {
    return 'c'
  }
}
```

Into this (to fit the style we are using in other places):

```js
if (a) {
  return 'a'
}
/* do stuff */
if (b) {
  return 'b'
}
return 'c'
```

Or even:

```js
if (a) {
  return 'a'
}
/* do stuff */
return b ? 'b' : 'c'
```

Reply:
ohhh yes sure!